### PR TITLE
Update roadmap.md

### PR DESCRIPTION
**Pull request by ewill00**
PR to update the roadmap.  Not including pictures at this point but taking out some things that we aren't going to get to soon and adding the change to separate out the Enrichment pattern from Ingestion

### CODE REVIEW

**Review by rguderian — APPROVED** [review]
Looks good to me

**Review by ranum — APPROVED** [review]

File: docs/roadmap.md [hunk 1]
```diff
@@ -1,52 +1,55 @@
...
+        - In case of errors within the bundle, individual resources are retried
+        - Errors are reported back to the data integrator via the kafka topic
 
 
-Here is what the implemented architecture looks like currently 
-![IngestionEnrichment2020](images/IngestionEnrichment2020.png)
+## Clinical Data Enrichment
+The _Clinical Enrichment_ health pattern has been separated from the _Ingestion_ pattern.  It is still available to be deployed and run as part of ingestion, but it can also be set up on its own.
+If it is run as part of _Ingestion_ it will Enrich the data as it comes into the environment, before it is stored in the FHIR Server.  It can also be run after the data is persisted in the FHIR server and configured to run on new or changed data.
+The goal for this _Enrichment_ pattern is that the enriched data will be written to a FHIR server by default, but because these patterns are just reference implementations, they can be modified and consumed in a number of ways.
```
Comment (ibmatc) [inline docs/roadmap.md:22]: Do we want to mention anything about the pattern using FHIR notifications or similar methods for triggering an enrichment to run?

**Review by ibmatc — APPROVED** [review]
LGTM - a couple minor comments but overall great improvements/clarifications.

File: docs/roadmap.md (hunk 1 above)
Comment (ewill00) [reply docs/roadmap.md:22]: I definitely want that detail to go into the github readme that describes the enrichment pattern when we have it.  Not sure whether to put that detail into the roadmap - I'll just leave it as  'configured to run on new or changed data' for now and we can update after it is fully delivered

File: docs/roadmap.md [hunk 2]
```diff
@@ -1,52 +1,55 @@
...
 
-At the end of 2020, the _Clinical Ingestion & Enrichment_ health pattern has made great progress as shown below.  This pattern is cloud agnostic (has been run on IBM Cloud, AWS, Azure, Google) and allows the following 
-- Take in HL7 or FHIR data from a source system (EMR, Data Warehouse, etc) via Kafka (preferred) or a simple API (mostly for testing)
-- Within a [NiFi](https://github.com/apache/nifi) canvas
+- Within the [NiFi](https://github.com/apache/nifi) canvas, the _Clinical Ingestion_ health pattern will:
     - Convert [HL7 to FHIR using technology from LinuxForHealth](https://github.com/LinuxForHealth/hl7v2-fhir-converter)
     - Validate the FHIR data without storing it in the FHIR Server
-    - Convert terminology using the FHIR Terminology Services
-    - Optionally [De-identify](https://github.com/Alvearie/de-identification) the FHIR data
+    - Put the converted/validated data on a Kafka topic for processing by the _Enrichment_ pattern
```
[the remark this answers is not in the document]
Comment (ewill00) [reply docs/roadmap.md:12]: Agree -  I think on the Roadmap that is ok to have it not fully reflect what is there today but where we are going.

**Pull request merged**